### ISSUE TITLE
Moment.js does not recognize some locales

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#8097]
**Describe the bug**
We use Moment.js to localize dates. However, this library does not recognize some of our locales. As a result, the date/time formats are not localized properly.

**To Reproduce**
1. Install and activate  the following language: `sr_RS@cyrillic`.
2. Go to Settings > Website > Setup > Date and Time.
3. See that the formatting options do not appear in cyrillic.

**What application are you using?**
OJS 3.3.0-11

**Additional information**
The following locale maps the PKP locale (key) to the recognized Moment.js locale (value):

```js
const map = {
    'sr_RS@latin': 'sr',
    'sr_RS@cyrillic': 'sr-cyrl',
    'uz_UZ@latin': 'uz-latn'
};
```

**Comment by NateWr** [comment]
The following PRs address this for `stable-3_3_0`. This will be fixed in `main` as part of work on #7191.

PR:
https://github.com/pkp/ui-library/pull/211

Tests:
https://github.com/pkp/ojs/pull/3461

**Comment by NateWr** [comment]
Merged to `stable-3_3_0`.
